app/bundles/PageBundle/Tests/Controller/PageControllerTest.php

```diff
@@ -45,6 +45,16 @@ protected function setUp(): void
         $this->id = $page->getId();
     }
 
+    /**
+     * Index should return status code 200.
+     */
+    public function testIndexAction(): void
+    {
+        $this->client->request('GET', '/s/pages');
```

Comment:
These kind of tests unfinished
You should check If you Landing pages page should return date created/modified column created by, and if modified date header contain icon with class fa-sort-amount-des